packages/core/src/sdk/product/useProductGalleryQuery.ts

```diff
@@ -66,20 +72,44 @@ export const useProductGalleryQuery = ({
     selectedFacets,
   })
 
+  const findFacetValue = (
+    facets: Facet[],
+    searchParam: string
+  ): string | null => {
+    const facet = facets.find(({ key }) => key === searchParam)
+    return facet?.value ?? null
+  }
+
   return useQuery<Query, Variables>(query, localizedVariables, {
     onSuccess: (data) => {
-      if (data && term) {
-        sendAnalyticsEvent<IntelligentSearchQueryEvent>({
-          name: 'intelligent_search_query',
-          params: {
-            locale,
-            term,
-            url: window.location.href,
-            logicalOperator: data.search.metadata?.logicalOperator ?? 'and',
-            isTermMisspelled: data.search.metadata?.isTermMisspelled ?? false,
-            totalCount: data.search.products.pageInfo.totalCount,
-          },
-        })
+      if (data) {
+        const fuzzyFacetValue = findFacetValue(selectedFacets, 'fuzzy')
+        const operatorFacetValue = findFacetValue(selectedFacets, 'operator')
+
+        if (!fuzzyFacetValue && !operatorFacetValue) {
+          setState({
```

Comment:
I noticed that the page is rendering twice. 

PLP: 
1. First load: /office
2. second load: /office?category-1=office&fuzzy=0&operator=and&facets=category-1%2Cfuzzy%2Coperator&sort=score_desc&page=0

Is there a way to set it without rendering twice?

Reply:
I've tried to solve this issue, but couldn't find a workaround for it :/ 

The team agreed to move forward with the upon approach and revisit it later.